ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouter.java

```diff
@@ -469,11 +475,22 @@ private void onResponse(List<ResponseInfo> responseInfoList) {
      */
     @Override
     public void run() {
+      long startTime;
+      long lastTime;
       try {
         while (isOpen.get()) {
+          startTime = time.milliseconds();
+          lastTime = startTime;
           List<RequestInfo> requestInfoList = pollForRequests();
+          routerMetrics.operationControllerPollforRequestTimeMs.update(time.milliseconds() - lastTime);
+          lastTime = time.milliseconds();
           List<ResponseInfo> responseInfoList = networkClient.sendAndPoll(requestInfoList);
+          routerMetrics.numOfRequestInfosPerPoll.update(requestInfoList.size());
+          routerMetrics.operationControllerSendAndPollTimeMs.update(time.milliseconds() - lastTime);
```

Comment:
Some more thought here:
If we have class A and class B calling their respective `NetworkClient`, and say class A has high request rate and class B request rate is low. Eventually we want to see that `sendAndPoll()' for class A has short polling time but not for class B. If we only tracks time for `sendAndPoll()` at network client level, the metric will not give us differentiated information between class A and B.

Though it is admitted that now only router is using network client.